src/containers/DataTable/index.js

```diff
@@ -29,6 +29,7 @@ export function DataTable ({
 	buttonWithAuthz, sort, noItemsComponent,
 	customButton, customComponent,
 	customRowStyle, customRowClassName,
+	customDescription,
```

Comment:
@titkov49 Maybe it would be better to call it `customCardBodyComponent` ? Or I suppose its a component, right? So developer can add not only a text component( a paragraph for instance) but also a button or something else, right?

Also I miss a documentation about how to use it